src/postgres/src/test/thread/thread_test.c

```diff
@@ -157,6 +158,7 @@ main(int argc, char *argv[])
 		 * 1 that uses it, so avoid using it.
 		 */
 		fprintf(stderr, "Failed to create thread 2 **\nexiting\n");
+        pthread_mutex_unlock(&init_mutex);
```

Comment:
Please align pthread_mutex_unlock() with the rest of the code (left indent 2 chars).